### ISSUE TITLE
Dynamic flags not functioning as intended

### ISSUE THREAD

**Issue by DanielBritze** [aau-network-security/haaukins#190]
Issues with the dynamic flags vary. Sometimes a flag work for the wrong exercise and sometimes it works for multiple exercises. Some team names with this issue is given:
```
Aars HTX | 21/11/2018 |   | frank dae | 
Aars HTX | 21/11/2018 |   | daniel kuhdal | 
Aars HTX | 21/11/2018 |   | emil |
Aars HTX | 21/11/2018 |   | dav | 
Aars HTX | 21/11/2018 |   | jle2myx | 
```

**Comment by mrturkmen06** [comment]
Apparently, the issue that Kaspar mentioned remains same. Hence, having a fix to this issue could be first priority.

**Comment by mrturkmen06** [comment]
Having merge of the [fix-dynamic-flags-#190](https://github.com/aau-network-security/haaukins/tree/fix-dynamic-flags-%23190) branch will resolve the issue, tested and it works as expected, there is such an issue that network scanning flag can be used for another exercises or vice versa. Static and dynamic flags work correctly. No need to have translated flag and I do not understand of having such a function, it messes up flags.

**Comment by mrturkmen06** [comment]
After merging of [fix-dynamic-flags-#190](https://github.com/aau-network-security/haaukins/tree/fix-dynamic-flags-%23190), the issue will be closed.

**Comment by mrturkmen06** [comment]
The main reason of having problems regarding to dynamic flags is directly related with this [translatedFlagForTeam](https://github.com/aau-network-security/haaukins/blob/0544e21ed5b0d6b37ce77bf1b0a3213b25612b35/svcs/ctfd/interception.go#L440) function, I totally did not get it. At first glance, I thought  that it is used for creating unique dynamic flags for per team however it is not the case, it makes everything messed up. For instance, when you enter any value to a challenge which has static flag, then it will show up "correct".  

When the function content is followed and investigated, it does not make any sense to me, I prefer to remove it. I did not see any valuable reason of having that function after all.

**Comment by mrturkmen06** [comment]
Fixed with new release [1.6.4](https://github.com/aau-network-security/haaukins/releases/tag/1.6.4)
